act-rules-format.md

```diff
@@ -206,6 +218,84 @@ For each accessibility requirement in the mapping, an ACT Rule <em class="rfc211
   </ul></blockquote>
 </aside>
 
+#### Secondary Requirements #### {#secondary-requirements}
+
+A secondary accessibility requirement is a requirement that is correlated with the rule, but for which the rule is not designed to test. The outcome of the rule impacts the result of the accessibility requirement, but the rule is not intended to test the conformance of that requirement. This correlation often results in some of the rule's test cases not satisfying the secondary accessibility requirement.
```

Comment:
I find this paragraph confusing, and don't think it helps in understand what is a Secondary Requirement. 
The following paragraph is much better at defining a secondary requirement (i.e. failed outcomes require further testing).